Assets/!/Scripts/PlayerAudio.cs

```diff
@@ -0,0 +1,58 @@
+using System.Diagnostics;
+using UnityEngine;
+using utils;
+
+public class PlayerAudio : MonoBehaviour
+{
+    public AudioSource audioSource;
+    public Audio[] audioPlayer;
+    private AudioClip _currentClip;
+
+    void Start()
+    {
+        audioSource = GetComponent<AudioSource>();
+    }
+
+    public void Walk()
+    {
+        PlayAudio("Walk", 1f);
+    }
+
+    public void Crouched()
+    {
+        PlayAudio("Crouched", 0.5f);
+    }
+
+    public void Crawl()
+    {
+        PlayAudio("Crawl", 0.15f);
+    }
+
+    public void Run()
+    {
+        PlayAudio("Walk", 1f);
+    }
+
+    public void Jump()
+    {
+        PlayAudio("Jump", 1f);
+    }
+    public void PlayAudio(string clipName, float volume)
+    {
+        Audio? audioToPlay = System.Array.Find(audioPlayer, a => a.id == clipName);
+
+        if (audioToPlay == null || audioToPlay.Value.clip == null)
+        {
+            UnityEngine.Debug.LogWarning("Audio clip not found: " + clipName);
+            return;
+        }
+
+        _currentClip = audioToPlay.Value.clip;
+
+        audioSource.clip = _currentClip;
```

Comment:
nit: I see `_currentClip` is just used here, we can remove the attribute and just do this:
```suggestion
        audioSource.clip = audioToPlay.Value.clip;
```

Reply:
It's changed now

Reply:
We still need to remove the unused attribute